src/Resources/translations/messages.en.yaml

```diff
@@ -8,3 +8,8 @@ webgriffe_sylius_akeneo:
         imported_yes: Yes, at %date%
         imported_no: No
         error_message: Error message
+        enqueue: Add to the queue
+        enqueued_success: Product enqueued successfully
+        product_not_exist: Product not found
+        product_already_enqueued: Product already added to the queue
+        product_without_code: Product does not have code
```

Comment:
I'd hide to the user the fact that there's a queue. So:
* "Add to the queue" should be "Schedule Akeneo PIM import"
* "Product enqueued successfully" should be "Akeneo PIM product import has been successfully scheduled"
* "Product already added to the queue" should be "Akeneo PIM import for this product has been already scheduled before"